packages/wonder-blocks-data/util/__tests__/memory-cache.test.js

```diff
@@ -76,7 +76,7 @@ describe("MemoryCache", () => {
             };
 
             // Act
-            cache.store(fakeHandler, "options", {data: "data"});
+            cache.store<string, string>(fakeHandler, "options", {data: "data"});
```

Comment:
New flow isn't able to infer the type of the store so we have to give it a little help.